e2e_tests/tests/nightly/test_convergence.py

```diff
@@ -27,3 +27,59 @@ def test_cifar10_pytorch_accuracy() -> None:
             target_accuracy, len(trial_metrics["steps"]), validation_errors
         )
     )
+
+
+@pytest.mark.nightly  # type: ignore
+def test_mnist_tp_accuracy() -> None:
+    config = conf.load_config(conf.official_examples_path("mnist_tp/const.yaml"))
+    conf.set_max_steps(config, 20)
```

Comment:
blocking: you can remove the `conf.set_max_steps` call here since it is now redundant.